source/assets/public/index.css

```diff
@@ -130,12 +130,9 @@ body {
     line-height: 4rem;
     text-align: left;
   }
-    .breakpoint-phone .hero__slogan span {
-      font-size: 3rem;
-      line-height: 4rem;
-    }
-  .breakpoint-phone .hero__date {
-    font-size: .1rem;
```

Comment:
Yeah the `.breakpoint-phone .hero__slogan span` can be removed now. When we were using Young serif for the slogan there were weird kerning issues with the apostrophe, so i needed to surround it in a span. That condition was to make sure the apostrophe scaled down alongside the rest of the text. 


I'm not sure why the `hero__date` was so small in the small version. that can be removed no problem. :)